eventing/triggers/new-trigger-filters.adoc

```diff
@@ -0,0 +1,47 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="new-trigger-filters"]
+= New trigger filters
+include::_attributes/common-attributes.adoc[]
+:context: new-trigger-filters
+
+toc::[]
+
+:FeatureName: New trigger filters feature
+include::snippets/technology-preview.adoc[]
+
+The new trigger filters feature adds a new `filters` field to triggers that aligns with the filters API field defined in the CloudEvents Subscriptions API. It allows you to specify filter expressions, where each expression evaluates to `true` or `false` for each event. 
+
+The following example shows a trigger using the new filters field:
+[source,yaml]
+----
+apiVersion: eventing.knative.dev/v1
+kind: Trigger
+metadata:
+  name: my-service-trigger
+spec:
+  broker: default
+  filters:
+    - cesql: "source LIKE '%commerce%' AND type IN ('order.created', 'order.updated', 'order.canceled')"
+  subscriber:
+    ref:
+      apiVersion: serving.knative.dev/v1
+      kind: Service
+      name: my-service
+----
+
+The `filters` field contains an array of filter expressions, each evaluating to either `true` or `false`. If any expression evaluates to `false`, the event will not be sent to the subscriber. Each filter expression uses a specific dialect that determines the type of filter and the set of allowed additional properties within the expression.
+
+To enable new trigger filters, add the `new-trigger-filters: enabled` entry in the section of the `KnativeEventing` type in the operator config map as follows:
+[source,yaml]
+----
+apiVersion: operator.knative.dev/v1beta1
+kind: KnativeEventing
+spec:
+  config:
+    features:
+      new-trigger-filters: enabled
+----
+
+include::modules/triggers-supported-filter-dialects.adoc[leveloffset=+1]
+include::modules/triggers-conflict-current-filter-field.adoc[leveloffset=+1]
+include::modules/triggers-annotations.adoc[leveloffset=+1]
```

Comment:
why is this `triggers-annotations` page here? The content is not related to event filtering

Reply:
The section was present in the 1.15 version: https://knative.dev/docs/eventing/triggers/#trigger-annotations which is why I included it. Do you want me to remove this? @pierDipi @matzew

Reply:
Again. for 1.34 - we are shipping 1.14 bits (as documented in the release matrix)

Reply:
I will remove the triggers-annotations section

Reply:
I agree on the trigger-annotation not really relating to the content for filtering.

Reply:
Removed the section